src/js/core/MesosStream.js

```diff
@@ -0,0 +1,4 @@
+import { stream } from "mesos-client";
+
+export const MesosStreamType = Symbol("MesosStreamType");
```

Comment:
🎨  The const name rubs me the wrong way, but I'm ok to merge as is.

Reply:
![](https://media.giphy.com/media/xT39DagQTi3uoMoGxq/giphy.gif)

Reply:
![](https://media.giphy.com/media/DeJ2ifS2V2zlu/giphy.gif)